components/Players/DraggableQueue.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-list>
+    <draggable v-model="queue" v-bind="dragOptions" class="list-group">
+      <v-hover
+        v-for="(item, index) in queue"
+        :key="`${item.Id}-${getUuid()}`"
+        v-slot="{ hover }"
+        class="pa-0 ma-0"
+      >
+        <v-list-item ripple class="pa-0 ma-0" @click="onClick(index)">
+          <v-list-item-action
+            v-if="!hover"
+            class="list-group-item d-flex justify-center d-flex text-caption"
+            :class="{ 'primary--text': isPlaying(index) }"
+          >
+            {{ index }}
```

Comment:
This starts counting from 0. Humans usually start counting at 1 :yum:

Reply:
It'd also make sense to have this index change to some icon to indicate that playback is in progress on this item. The color already suggests it, but making it more explicit would be nice, I think.

Reply:
That's another PR with howler maybe